<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTabWidget" name="plot_tabWidget">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>29</y>
     <width>1031</width>
     <height>651</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::South</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab1">
    <attribute name="title">
     <string>總手術時間</string>
    </attribute>
    <widget class="QLabel" name="plot_label1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>930</width>
       <height>600</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>尚未引入數據!</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2">
    <attribute name="title">
     <string>總中斷時間</string>
    </attribute>
    <widget class="QLabel" name="plot_label2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>930</width>
       <height>600</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>尚未引入數據!</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab3">
    <attribute name="title">
     <string>總中斷次數</string>
    </attribute>
    <widget class="QLabel" name="plot_label3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>930</width>
       <height>600</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>尚未引入數據!</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab4">
    <attribute name="title">
     <string>中斷時間/總手術時間比例 </string>
    </attribute>
    <widget class="QLabel" name="plot_label4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>930</width>
       <height>600</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>尚未引入數據!</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="button_back_menu">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>630</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        border: 0px solid;
    }
    QPushButton:hover {
        background-color: rgb(223, 223, 223);
    }
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/back.png</normaloff>image/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>700</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>plot_tabWidget</zorder>
  <zorder>button_back_menu</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
